src/PodActions.jsx

```diff
@@ -61,46 +61,26 @@ const PodDeleteModal = ({ pod }) => {
     </Modal>;
 };
 
-export class PodActions extends React.Component {
-    static contextType = DialogsContext;
-
-    constructor(props) {
-        super(props);
-        this.state = {
-            isOpen: false
-        };
-        this.onToggle = isOpen => {
-            this.setState({
-                isOpen
-            });
-        };
-        this.onSelect = event => {
-            this.setState({
-                isOpen: !this.state.isOpen
-            });
-        };
-    }
+export const PodActions = ({ onAddNotification, pod }) => {
+    const Dialogs = useDialogs();
+    const [isOpen, setOpen] = useState(false);
 
-    render() {
-        const Dialogs = this.context;
-        const { isOpen } = this.state;
-        const pod = this.props.pod;
-        const dropdownItems = [];
-        // Possible Pod Statuses can be found here https://github.com/containers/podman/blob/main/libpod/define/podstate.go
-        if (pod.Status == "Running" || pod.Status == "Paused") {
-            dropdownItems.push(
-                <DropdownItem key="action-stop"
+    const dropdownItems = [];
+    // Possible Pod Statuses can be found here https://github.com/containers/podman/blob/main/libpod/define/podstate.go
+    if (pod.Status == "Running" || pod.Status == "Paused") {
+        dropdownItems.push(
+            <DropdownItem key="action-stop"
                               className="pod-action-stop"
                               onClick={() =>
                                   client.postPod(pod.isSystem, "stop", pod.Id, {})
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to stop pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1146-20221124-132403-fa425528-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)